tests/dummy/config/environment.js

```diff
@@ -48,12 +41,6 @@ module.exports = function(environment) {
 
     ENV.APP.rootElement = '#ember-testing';
     ENV.APP.autoboot = false;
-
-    ENV.contentSecurityPolicyMeta = true;
-    ENV.contentSecurityPolicy = {
-      'style-src': ["'self'", "'nonce-123456789'"]
-    };
-    ENV['ember-tableish-csp-nonce'] = '123456789';
```

Comment:
The eCSP package was throwing some weird errors running tests -- this probably isn't ideal long-term, as consuming apps using eCSP might get some errors. Might be able to debug and get it back in action

Reply:
does adding the removed config values back fix anything?

Reply:
Yeah, these I removed after not being able to get it work. I updated eCSP and still saw the issue. This is the closest thread I was able to find with the error I was getting:
https://github.com/rwjblue/ember-cli-content-security-policy/issues/127

Reply:
```
Global error: Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'nonce-abcdefg'"
```